tests/_data/TestedRoboFile.php

```diff
@@ -22,10 +22,10 @@ public function hello($name = null, $opts = ['yell' => false, 'to' => null]) {
      *     +--------+-------------+
      *
      * @param int $start Number to start from
-     * @param $steps int Number of steps to perform
+     * @param int $steps Number of steps to perform
```

Comment:
phpdocumentor/reflection-docblock does not allow type to come after the variable name. Since placing the type after the variable name is not supported in the proposed PSR-5 docblock standard, it's probably okay to encourage Robo commands to order their type names appropriately.  Those who do not will see their type name at the head of their argument's help text -- just a minor annoyance.